readme.md

```diff
@@ -153,6 +153,25 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
+## Certificates -  Important to know
+
+The Azure IoT Hub certificates presented during TLS negotiation shall be always validated using the appropriate root CA certificate(s).
+
+The samples in this repository leverage the certificates in `certs.c` for the United States, Germany sovereign cloud and China sovereign cloud.
```

Comment:
This specific sentence is not for our sdk. certs.c is not in our samples. Also samples for .net are in another repository.

Reply:
@abhipsaMisra - Is there any example Sejal point to for this?

Reply:
updated

Reply:
No, afaik we don't hardcode certificates anywhere in our samples -> one of our samples actually points to the c repo: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/provisioning/Samples/device/X509Sample